engines/python/setup/djl_python/huggingface.py

```diff
@@ -58,6 +59,11 @@ def initialize(self, properties: dict):
         # https://huggingface.co/docs/accelerate/usage_guides/big_modeling#designing-a-device-map
         if "device_map" in properties:
             kwargs["device_map"] = properties.get("device_map")
+            logging.info(f"Using device map {kwargs['device_map']}")
+        elif "tensor_parallel_degree" in properties:
+            kwargs["device_map"] = "auto"
+            world_size = torch.cuda.device_count()
+            logging.info(f"Using {world_size} gpus")
```

Comment:
We should use tensor_paralel_detree here. or we should add a check to verify world_size and tensor_paralel_detree are equal?

Reply:
added a check here